SOOP/SOOP_ASF_SST_XBT/soop_xbt_pipeline/create_plot.py

```diff
@@ -7,32 +7,17 @@
 
 import datetime
 import itertools
-import os
-import re
-import shutil
-import string
 import sys
 import tempfile
 
-import matplotlib
 import matplotlib.pyplot as plt
-import pylab
-from matplotlib.pyplot import figure, plot, show, title, xlabel, xticks, ylabel
+from matplotlib.pyplot import plot, title, xticks
```

Comment:
Since the purpose of this PR is "code cleaning", you could also make the use of pyplot functions more consistent, i.e. *either*
  `import matplotlib.pyplot as plt` and then use `plt.plot`, `plt.title` etc...
*or*
  `from matplotlib.pyplot import figure, plot, show, title, ` etc... for all pyplot functions.

Not essential, but cleaner (IMHO)